phpBB/phpbb/controller/helper.php

```diff
@@ -143,6 +144,15 @@ public function route($route, array $params = array(), $is_amp = true, $session_
 		$context = new RequestContext();
 		$context->fromRequest($this->symfony_request);
 
+		if ($this->config['force_server_vars'])
+		{
+			$context->setHost($this->config['server_name']);
+			$context->setScheme(substr($this->config['server_protocol'], 0, -3));
```

Comment:
actually here we are just removing the 3 last characters because in the acp we require `http://` or `https://`